python/lsst/sims/catUtils/exampleCatalogDefinitions/phoSimCatalogExamples.py

```diff
@@ -9,14 +9,69 @@
 
 __all__ = ["write_phoSim_header", "PhosimInputBase",
            "PhoSimCatalogPoint", "PhoSimCatalogZPoint",
-           "PhoSimCatalogSersic2D", "PhoSimCatalogSSM", "PhoSimSpecMap"]
+           "PhoSimCatalogSersic2D", "PhoSimCatalogSSM", "PhoSimSpecMap",
+           "DefaultPhoSimHeaderMap"]
 
 
 PhoSimSpecMap = SpecMap(fileDict=defaultSpecMap.fileDict,
                         dirDict={'(^lte)': 'starSED/phoSimMLT'})
 
 
-def write_phoSim_header(obs, file_handle):
+# This is a dict of transformations mapping from data in OpSim to header
+# values expected by PhoSim.  The dict is keyed on the names of PhoSim
+# parameters.  The values of the dict are either straight values, in which
+# case those values are written to the PhoSim InstanceCatalog header, or tuples
+# containing the name of the OpSim column which should be use to calculate
+# the PhoSim parameter and any transformation needed to go from OpSim
+# units to PhoSim units.
+DefaultPhoSimHeaderMap = {'rottelpos': ('rotTelPos', np.degrees),
```

Comment:
Maybe a url here to the phosim header information, not sure if it's already elsewhere in the documentation.